.github/workflows/build.yml

```diff
@@ -62,6 +62,19 @@ jobs:
       if: matrix.cxx == 'g++-10' && startsWith(github.ref, 'refs/tags/')
       id: hash-gcc
       run: echo "::set-output name=hashes::$(sha256sum Linux.flatc.binary.${{ matrix.cxx }}.zip | base64 -w0)"
+      
+  build-linux-no-file-tests:    
+    name: Build Linux with -DFLATBUFFERS_NO_FILE_TESTS
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        cxx: [g++-10, clang++-12]
```

Comment:
We can probably skip the matrix of compilers and just choose clang++12

Reply:
Sounds good. Done.